docker/README.md

```diff
@@ -45,3 +45,17 @@ You will see it give a forwarding address like this one:
 You may need to temporarily set your `siteurl` and `home` `wp_option`s to the new url. You can do this with phpMyAdmin or WP-CLI.
 
 Visit the `<url>` , login and setup WCPay.
+
+### Changing default port for xDebug
+To change the default port for xDebug you should create `docker.compose.override.yml` with the following contents:
```

Comment:
I think that the file name should be `docker-compose.override.yml` (tested with this filename and it worked and it is in the `.gitignore`, so it doesn't get committed to the repo!)

Reply:
You're totally right, great catch! Fixed in https://github.com/Automattic/woocommerce-payments/pull/5269/commits/5c315495fa5b0f7b443506178a9fdebe24b2cf7d.